src/plugins/stake-plugins/generic/pluginInfo/tarotUtils.ts

```diff
@@ -257,6 +257,40 @@ export const tarotUtils = (config: TarotPool['adaptor'], provider: ethers.provid
         bAmountBMin: bAmountB / Math.sqrt(SLIPPAGE_FACTOR)
       }
     },
+    async getNewLeverage(changes: typeof this.NO_CHANGES): Promise<number> {
+      const { valueCollateral, valueA, valueB } = await this.getValues(changes)
+      const valueDebt = valueA + valueB
+      if (valueDebt === 0) return 1
+      const equity = valueCollateral - valueDebt
+      if (equity <= 0) return Infinity
+      return valueDebt / equity + 1
+    },
+    async getLeverage(): Promise<number> {
+      return await this.getNewLeverage(this.NO_CHANGES)
+    },
```

Comment:
Is "leverage" a factor or a percentage? Like 500% or 5x? Can we add a comment to clarify what leverage is and how it's calculated?

Reply:
it's copied from tarot.to